arangod/Agency/Node.h

```diff
@@ -286,26 +176,13 @@ class Node final {
   /// @brief Is object
   bool isObject() const;
 
-  /**
-   * @brief Set expiry for this node
-   * @param Time point of expiry
-   */
-  void timeToLive(TimePoint const& ttl);
-
   /// @brief accessor to Node object
   /// @return  returns nullptr if not found or type doesn't match
-  Node const* hasAsNode(std::string const&) const noexcept;
-
-  /// @brief accessor to Node object
-  Node& hasAsWritableNode(std::string const&);
-
-  /// @brief accessor to Node's type
-  /// @return  returns nullopt if not found or type doesn't match
-  std::optional<NodeType> hasAsType(std::string const&) const noexcept;
+  std::shared_ptr<Node const> hasAsNode(std::string const&) const noexcept;
```

Comment:
This is the same as `get` with a string argument, we should remove this one here and use `get` directly instead.